Frameworks/CoreGraphics/CGPath.mm

```diff
@@ -626,29 +644,20 @@ void CGPathApply(CGPathRef path, void* info, CGPathApplierFunction function) {
 }
 
 /**
- @Status Interoperable
- @Notes
+ @Status Caveat
+ @Notes eoFill ignored. Default fill pattern for ID2D1 Geometry is used.
 */
 bool CGPathContainsPoint(CGPathRef path, const CGAffineTransform* transform, CGPoint point, bool eoFill) {
     if (transform) {
         point = CGPointApplyAffineTransform(point, *transform);
     }
-    // check if the point is outside this box already, if it is, return false
-    CGRect boundingBox = CGPathGetBoundingBox(path);
-    if (!CGRectContainsPoint(boundingBox, point)) {
-        return false;
-    }
 
-    CGContextRef context =
-        CGBitmapContextCreate(0, boundingBox.origin.x + boundingBox.size.width, boundingBox.origin.y + boundingBox.size.height, 1, 1, 0, 0);
+    BOOL containsPoint = false;
```

Comment:
Although it should be initialized to FALSE to match the type.

---
In reply to: [85579045](https://github.com/Microsoft/WinObjC/pull/1255#discussion_r85579045) [](ancestors = 85579045,85578185)